### PR TITLE
#3634 - Change student account restriction content

### PR DESCRIPTION
**Pull request by katrina-aot-git**
image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/studentaccountactivity.json
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/studentaccountactivity.json:1]: Please review the below AC about "Code UNCL", it should reads "Code UNCL - Maximum funding of unclassified funding"

Comment (andrewsignori-aot) [reply sources/packages/forms/src/form-definitions/studentaccountactivity.json:1]: Please review it, there is an extra white spacing in the sentence.

Comment (katrina-aot-git) [reply sources/packages/forms/src/form-definitions/studentaccountactivity.json:1]: found the extra space

File: sources/packages/forms/src/form-definitions/studentaccountactivity.json [hunk 1]
```diff
@@ -8813,7 +8843,8 @@
                   "addons": [],
                   "tree": false,
                   "lazyLoad": false,
-                  "id": "eo9ugh4"
+                  "id": "eo9ugh4",
+                  "isNew": false
                 },
                 {
                   "title": "Code IS -  Canada Student Financial Assistance Program restriction",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/studentaccountactivity.json:8850]: Please remove the extra white spacing.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for making the changes and for the extensive content change. Looks good 👍

**Pull request merged**